test/acceptance/ember-cli-update-test.js

```diff
@@ -401,6 +401,7 @@ describe(function() {
       commitMessage: 'my-app',
       install: true,
       addon: location,
+      blueprint: 'ember-cli-update-git-addon-test',
```

Comment:
This test is no longer testing a default blueprint?

Reply:
Modified code to ensure default blueprint https://github.com/ember-cli/ember-cli-update/pull/1084/commits/8579ff6a9d1b1207e3ec1953e25194556dba79f0 installed if no override is passed 

Also added code to accommodate https://cli.emberjs.com/release/writing-addons/addon-blueprints/#blueprintconfig